### DISCUSSION TITLE
Possibly issue with new beta (and soon to be live) emby server

### DISCUSSION THREAD

**Discussion by vw-kombi** [arabcoders/watchstate#394] [General] [answered]
I love your watchstate and it runs daily for my own user to keep my primary emby server and my hot standby emby server in sync.
For the other users, I use the emby backup/restore plugin they have and manually select all users except mine (as they are not so important).

Emby devs however have implemented a change on the latest .66 beta that replaces this plugin with a new 'all in' backup system - which removes the abiity to 'cherry pick' what you want to restore - it is a complete copy of the server - inc server name etc etc.  This is due to go live this month for everyone I believe.

I posted that this will not work for me, and they have said the move to the database for almost all of the emby stuff will meand it cant be done this way.

I am just posting this here in case they also break the method you are using to sync users.
https://emby.media/community/index.php?/topic/124566-48066-backup-plugin-revamp/

**Comment by vw-kombi** [comment]
After back and forth with the emby devs, they have agreed to provide the original emby backup plugin which has only the user restore process - so maybe no issue for watchstate functionality after all.  It is posted in page1 of the above link.  Note - you can only see these links if you are part of the emby beta users.

**Comment by arabcoders** [comment, marked as answer]
Hi, 

Thank you for bringing this up, Thankfully we don't use this particular functionality of emby. 

We use the API directly. 

And you could also use watch state to sync your other users as well just mount different data dir for each user and run import&export you don't have to run 25 container those users can be synced manually using cronjob.  https://github.com/arabcoders/watchstate/issues/136#issuecomment-1145767663
